trlx/orchestrator/ppo_orchestrator.py

```diff
@@ -193,29 +193,34 @@ def make_experience(self, num_rollouts: int = 1024, iter_count: int = 0):  # noq
             ref_logprobs = ref_logprobs.cpu()
             query_tensors = query_tensors.cpu()
             response_tensors = response_tensors.cpu()
+
             if self.trainer.config.model.model_arch_type == "seq2seq":
                 start = 1  # skip the <s> token
                 ends = (response_tensors[:, start:] != 0).sum(1)
-                all_logprobs = [logprobs[ix, start : ends[ix]] for ix in range(n)]
-                all_values = [values[ix, start - 1 : ends[ix] - 1] for ix in range(n)]
+
```

Comment:
This errors out as all_logprobs has been removed but is used regardless in PPORLElement